### ISSUE TITLE
Render ETCS location markers

### ISSUE THREAD

**Issue by bigbug21** [OpenRailwayMap/OpenRailwayMap#755]
ORM should be able to render ETCS location markers. They are to replace the old-style "Blockkennzeichen" in German. An image of these new style of "Blockkennzeichen" is available at https://de.wikipedia.org/wiki/Blockkennzeichen#/media/Datei:Blockkennzeichen_alleinstehend.jpg.

**Comment by DerDakon** [comment]
Right, but we first need to define a proper tagging for this. I asked about that on the OpenRailwayMap mailing list, let's see where it goes.

**Comment by checkxp** [comment]
Please don't confuse the ETCS "Stop markerboards" and "Location markerboards".
The one showed in the picture of the german Wikipedia article is only a "location markerboard" and isn't used in every country installing ETCS.
In a lot of implementaiton of ETCS, only the stop markerboards will be used. 

See the ERA speicifications for the Stop Markerboards: https://www.era.europa.eu/sites/default/files/filesystem/ertms/ccs_tsi_annex_a_-_mandatory_specifications/set_of_specifications_1_etcs_b2_gsm-r_b1/index038_-_06e068_v20.pdf

Here is a brief explanation of the difference between both: https://www.era.europa.eu/sites/default/files/activities/docs/harmonised_ertms_marker_boards_en.pdf

For signaling experts, these are the difference between an EOA protecting a dangerpoint (non-overridable EOA) and one for block-section separation EOA (in some countries seen as overridable EOA).
